### ISSUE TITLE
[question] mapping directory

### ISSUE THREAD

**Issue by Svnipni** [bactopia/bactopia#464]
Hello Robert!

I'm having a lot of fun with this pipeline. Right now I'm trying to have bactopia output the mapping files (.bam) too. 
Thusfar, I hadn't really needed it but I'm trying out a few new tools.

I must be missing something of course. What parameters do I need to provide for bactopia to output and keep the mapping .bam files?

Cheers!

**Comment by rpetit3** [comment]
Hi @Svnipni 

Glad you are having fun! Are you wanting to get BAM files based alignments to a reference?

Cheers,
Robert

**Comment by Svnipni** [comment]
Hi Robert!
No, preferably aligned to the assembled genome. Amongst other things, I mean to wrap several samples up to construct a pangenome with anvio. If I understand correctly, it needs the .bam file mapped to the fasta and contig.db it constructs.

**Comment by rpetit3** [comment]
If you are using Illumina data one thing you could consider is running [Snippy](https://bactopia.github.io/v3.0.0/bactopia-tools/snippy/) with `--skip_phylogeny` then make use of the [BAM files output from Snippy](https://bactopia.github.io/v3.0.0/bactopia-tools/snippy/#snippy).

**Comment by Svnipni** [comment]
Thank you!
I tried running snippy before, but it didn't recognize the genbank reference file downloaded from NCBI.

```
echo "ERROR: Reference file (GCA_012767755.2.gb) does not appear to be a GenBank file"
      exit 1

```
Just for giggles, I tried it with the annotated .gbk.gz output from bactopia but got the same error.

**Comment by Caitgwill** [comment]
I ran into this issue with snippy. Snippy doesn't like zipped reference files. Unzip the gbk file and it should work.

**Comment by rpetit3** [comment]
Should we fix this? In other words, should a compressed Genbank file be an allowed input? (I personally think it should be, and am failing to remember why I didn't allow it.)

**Comment by Caitgwill** [comment]
Yes, I think it would be good to allow compressed gbk and fna files as inputs.

**Comment by rpetit3** [comment]
Hi everyone, 

I don't remember when I added it, but the snippy tool will now allow uncompressed and compressed reference files.

Cheers,
Robert
